src/Paket.VisualStudio/Utils/PaketLauncher.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Diagnostics;
+
+namespace Paket.VisualStudio.Utils
+{
+    public static class PaketLauncher
+    {
+        public static void LaunchPaket(string SolutionDirectory, string PaketSubCommand, DataReceivedEventHandler PaketDataReceivedHandler)
+        {
+                Process process = new Process();
+                process.StartInfo.FileName = SolutionDirectory + ".paket\\paket.exe";
```

Comment:
what we should do: if the ".paket\\paket.exe" is not there, then try to call ".paket\\paket.bootstrapper.exe"

Reply:
I had assumed that as a configuration step and would be already done by the user but if it is important, I can update the PR to call .paket\paket.bootstrapper.exe first or make another patch for it if this one gets merged.

Reply:
yes please call pakt.bootstrapper.exe if and only if paket.exe is not there

if both are not there then please emit proper error

Reply:
I have updated the code with the desired changes